i18n/locales/source/pegasus/mobile.yml

```diff
@@ -632,18 +632,12 @@
   dashboard_course_csp: "AP® Computer Science Principles"
   dashboard_course_csp_desc: "Code.org has designed a rigorous, engaging, and approachable course that explores the foundational ideas of computing."
   dashboard_landing_loading: "Loading..."
-  dashboard_action_edit: "Edit"
```

Comment:
@caleybrock Just confirming these student management strings are removed intentionally? I assume because this table is no longer part of pegasus but now part of dashboard? Which file did we move these strings to?

Reply:
Intentionally moved to apps: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/i18n/common/en_us.json#L398

Reply:
I did a search of the code base before removing them to make sure they weren't used elsewhere.